packages/ui/src/modal/list/ListModal.tsx

```diff
@@ -89,41 +78,18 @@ type Props = Pick<Parameters<typeof Modal>['0'], 'trigger'> & {
 }
 
 const Image = styled('img', {})
-const Span = styled('span', {})
-const ContentContainer = styled(Flex, {
-  width: '100%',
-  borderColor: '$borderColor',
-  flexDirection: 'column',
-  '@bp1': {
-    flexDirection: 'row',
-  },
-})
-
-const MainContainer = styled(Flex, {
-  flex: 1,
-  borderColor: '$borderColor',
-  borderTopWidth: 1,
-  borderLeftWidth: 0,
-  '@bp1': {
-    borderTopWidth: 0,
-    borderLeftWidth: 1,
-  },
-
-  defaultVariants: {
-    direction: 'column',
-  },
-})
 
 const MINIMUM_AMOUNT = 0.000001
 
+const minimumDate = dayjs().add(1, 'h').format('MM/DD/YYYY h:mm A')
```

Comment:
Does this need to be recalculated every time the modal is opened? In the case that they browse the site for an hour

Reply:
Hm this is how we did it on the BidModal. That's a good point though